AV-45749: Update sample queries (#35)
couchbaselabs/docs-devex: modules/n1ql/pages/n1ql_application_continuity.adoc

Reviewer: I think I dotted this page as it doesn't really apply.

Author: This page is dotted in this branch too?

Author: File got duplicated somewhere and was undotted, the duplicate has been removed.

Reviewer 2: This file should be dotted for Elixir as it is in Capella.

Author: This is weird. That page is dotted in my local repo but it doesn't show up here

Author: File history says it was pulled directly from the Capella branch, I don't know where it got undotted. I don't know how to change it, it's dotted on my end but doesn't show up as being changed.

Author: Duplicate file removed

= {sqlpp} Application Continuity

> :description: No special strategy is required to ensure {sqlpp} application continuity during Couchbase Server upgrades if you are already running Couchbase Server 5.0 or later.

Reviewer 3: ```suggestion
:description: If you're already running Couchbase Server 5.0 or later, you don't need to do anything for {sqlpp} application continuity during an upgrade. 
```

{description}

> If you are on a pre-5.0 release, special care must be taken to ensure {sqlpp} application continuity if 5.x is part of your upgrade path.

Reviewer 3: ```suggestion
If you are on a pre-5.0 release and 5.x is a part of your upgrade path, you must take some additional steps for {sqlpp} application continuity.
```

> Refer to xref:install:upgrade.adoc[Upgrading Couchbase Server] for more information about upgrade paths.

Reviewer 3: ```suggestion
For more information about upgrade paths, see xref:install:upgrade.adoc[Upgrading Couchbase Server].
```
